core-services/image-mirroring/stackrox/mapping_stackrox_collector_drivers

```diff
@@ -0,0 +1 @@
+brew.registry.redhat.io/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 registry.ci.openshift.org/stackrox/stackrox-drivers:0.1.0
```

Comment:
you are mirroring to `registry.ci.openshift.org`, but are using push creds for `brew`. Is this what you are intending to do? If you just need to mirror to our registry you should add this file to https://github.com/openshift/release/tree/5b427fd6abdd1b015666f2471140a8ef9a29a568/core-services/image-mirroring/supplemental-ci-images, and use the existing job.

Reply:
I think I may have been misguided in thinking `brew.registry.redhat.io` was a private registry and as such it would require dedicated credentials. I'll move it into the supplemental-ci file for stackrox images.